src/factory.js

```diff
@@ -284,6 +285,7 @@ export default function plotComponentFactory(Plotly) {
     style: PropTypes.object,
     className: PropTypes.string,
     useResizeHandler: PropTypes.bool,
+    chartId: _propTypes2.default.string
```

Comment:
Thanks for this PR! If you can change this to `PropTypes` from `_propTypes2` and rename the prop to `divId` I'd happily merge this and release it tomorrow!